server/src/main/java/org/elasticsearch/action/search/TransportSearchAction.java

```diff
@@ -142,7 +145,8 @@ public TransportSearchAction(
         ActionFilters actionFilters,
         IndexNameExpressionResolver indexNameExpressionResolver,
         NamedWriteableRegistry namedWriteableRegistry,
-        ExecutorSelector executorSelector
+        ExecutorSelector executorSelector,
+        TracingPlugin.Tracer tracer
```

Comment:
Should we add this here? If we start by tracing tasks, we won't need this dependency in the search action.

Reply:
It at least demonstrates that we're creating and registering it when creating the `Node`. We can drop it when we use the injected thing elsewhere.

Reply:
> It at least demonstrates that we're creating and registering it

Yes, the purpose was to make the `Tracer` instance available for the next steps, whatever/wherever they will be